app/samples/src/main/res/values/styles.xml

```diff
@@ -22,4 +46,17 @@
     <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar" />
     <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
 
+    <style name="AboutDialogItem">
+        <item name="android:textSize">16sp</item>
+        <item name="android:layout_marginTop">4dp</item>
```

Comment:
nebolo by lepsie (univerzalnejsie), keby bol `top = 2` a `bottom = 2`? alebo nechceme, aby posledny bol 2dp od spodku? aj tak je tam asi padding

Reply:
zkoušel jsem teď tu variantu co navrhuješ a nějak to v tom ConstraintLayout-u ignoruje ten bottom margin...

Reply:
bottom toho posledneho, ci vsetkych?

Reply:
všech :(

Reply:
fakt? top funguje a bottom nie? 🤔 divne

Reply:
mohu ukázat, asi dělám něco zle, nevím :(